art/attacks/evasion/adversarial_patch/adversarial_patch_tensorflow.py

```diff
@@ -495,3 +496,17 @@ def reset_patch(self, initial_patch_value: Optional[Union[float, np.ndarray]] =
             self._patch.assign(initial_patch_value)
         else:
             raise ValueError("Unexpected value for initial_patch_value.")
+
+    @staticmethod
+    def insert_transformed_patch(x: np.ndarray, patch: np.ndarray, image_coords: np.ndarray):
+        """
+        Insert patch to image based on given or selected coordinates.
+
+        :param x: The images to insert the patch.
```

Comment:
I think this function is only applied for a single image, so this description confuses me that the patch can be applied for a batch of images.